.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Bug Report
+about: Something isn't working as expected? Create a report to help us improve.
+labels: "c.Bug"
+---
+
+<!--
+  Before opening a new issue, please search existing issues:  https://github.com/reposense/RepoSense/issues
+-->
+
+**Tell us about your environment**
+
+* **RepoSense Version:**
+
+**What did you do? Please include the steps to reproduce the bug.**
+
+
+**Add the actual source code causing the issue.**
```

Comment:
Is this question asking the bug reporter to identify problematic code within the RepoSense codebase?

Reply:
I think we should keep this optional explicitly ( in case the bug reporter is just a user )

Reply:
It can be changed to 
**If possible, add the actual source code causing the issue.**

Reply:
Should we move this question to the end of the template? Since it's only not relevant for all bug reporters?